package.json

```diff
@@ -30,6 +30,7 @@
     "test:unit": "yarn test:karma && yarn test:server && yarn test:browser",
     "posttest:e2e": "yarn stop",
     "test:e2e": "yarn --cwd test/e2e start",
+    "test:e2e:tokens": "export REFRESH_TOKEN=true && yarn test:e2e",
```

Comment:
If `REFRESH_TOKEN` is set, test runner will pick a wdio config file that runs only refresh token related specs.